src/app/progress/TrackedTriumphs.tsx

```diff
@@ -32,6 +33,25 @@ export function TrackedTriumphs({
     );
   }
 
+  // determine absolute path of record for sorting purpose
+  const record_path = (r: DestinyRecordDefinition) => {
+    const path: string[] = [];
+    let parent: (DestinyRecordDefinition | DestinyPresentationNodeDefinition) = r;
+
+    while(parent?.parentNodeHashes?.length > 0) {
+      path.unshift(parent.displayProperties.name)
+      parent = defs.PresentationNode.get(parent.parentNodeHashes[0]);
+    }
+    return path;
+  };
+
+  // sort by parent node groups (alphabetically)
+  records = records.sort((a,b) => {
+    const pathA = record_path(a.recordDef).join("/")
+    const pathB = record_path(b.recordDef).join("/")
+    return pathA.localeCompare(pathB);
+  })
```

Comment:
```suggestion
  records = _.sortBy(records, ((record) => recordPath(record.recordDef).join("/"));
```

This will avoid repetitively computing the path string as different pairs are compared.

Reply:
changed to use _.sortBy as recommended